### ISSUE TITLE
This link seems no longer available

### ISSUE THREAD

**Issue by NaturalSelect** [LinWin-Cloud/Setool-Main#5]
https://github.com/LinWin-Cloud/Setool-Main/blob/fb1b00d14df9285f087f73910f1465cc4f50dbf5/search_hacker.java#L81

do you mean 'https://google.com/'?

**Comment by LinWin-Cloud** [comment]
Yes,this link mean google's website. But in China Mainland can not use google search and service.So this is a mirror website that all the people in world can visit google search.

Google hacker is a mainly part of the social enginning attack.

Thank for your issues.
